README.md

```diff
@@ -1,23 +1,27 @@
 # OPC-UA Device Service
 
 ## Overview
+
 This repository is a Go-based EdgeX Foundry Device Service which uses OPC-UA protocol to interact with the devices or IoT objects.
 
-## Feature
+## Features
 
-1. Subscribe data from OPCUA endpoint
+1. Subscribe/Unsubscribe one or more variables (writable configuration)
 2. Execute read command
-2. Execute write command
+3. Execute write command
+4. Execute method (using Read command of device SDK)
+
+## Prerequisites
 
-## Prerequisite
-* MongoDB
-* Edgex-go: core data, core command, core metadata
-* OPCUA Server
+- Edgex-go: core data, core command, core metadata
+- OPCUA Server
 
 ## Predefined configuration
 
-### Pre-define Devices
-Define devices for device-sdk to auto upload device profile and create device instance. Please modify `configuration.toml` file which under `./cmd/res` folder
+### Pre-defined Devices
+
+Define devices for device-sdk to auto upload device profile and create device instance. Please modify `devices.toml` file found under the `./cmd/res/devices` folder
```

Comment:
if the pre-defined device and profile is provided, the instruction of setting up the simulator should also be provided

Reply:
Hmm.. I've been using a simulator developed internally for testing. I'll look into testing with another simulator so I can provide this information.

Reply:
Updated readme to specify the Prosys simulation server, and updated the default profile to use pre-defined nodes.